cmake/FindMBEDTLSSO.cmake

```diff
@@ -0,0 +1,19 @@
+find_library(MBEDTLSSO_LIBRARY
```

Comment:
I dont think there is a need for 2 different FindMbedTLS files -- one should be enough